DEVEL.md

```diff
@@ -209,14 +209,26 @@ $ git clone https://github.com/<GitHub Username>/community-operators
 
 Generate package manifests with:
 ```
-$ make packagemanifests OPERATOR_VERSION=0.X.Y
+$ make bundle OPERATOR_VERSION=0.X.Y
+```
+
+Build bundle image and push to a local docker registry:
+```bash
+$ cd community-operators/operators/intel-device-plugins-operator/0.X.Y/ && docker build . -t <Local IP Address>:5000/intel-device-plugins-operator:0.X.Y
+$ docker run -d -p 5000:5000 --restart=always --name registry registry:2
+$ cat /etc/docker/daemon.json # Add (or create) the following line to /etc/docker/daemon.json
+{
+  "insecure-registries" : ["<Local IP Address>:5000"]
+}
+$ systemctl restart docker
```

Comment:
I tried to follow the steps in this PR but I did not need this. Do you use docker as the runtime in your cluster?

Reply:
Ah, yeap. Actually only in case docker is used.! would I add to the comment it is for docker user?

Reply:
Umm, is it related to "runtime"? This process is for pushing an image to a registry..!